### PR TITLE
Upgrading dependencies.

### PR DESCRIPTION
**Pull request by dessalines**
Still in draft, since `className={...}` isn't working yet.

### CODE REVIEW

File: fuse.ts [hunk 1]
```diff
@@ -36,9 +36,8 @@ const fuseServerOptions: FuseBoxOptions = {
 };
 Sparky.task("clean", () => {
    /**Clean distribute (dist) folder */
-   Sparky.src("dist")
-      .clean("dist")
-      .exec();
+   Sparky.src("dist/")
+      .clean("dist/")
```
Comment (dessalines) [inline fuse.ts:40]: Removing exec made this reliable.

File: package.json [hunk 2]
```diff
@@ -5,32 +5,34 @@
       "test": "node -r ts-node/register --inspect fuse.ts test"
    },
    "dependencies": {
+      "classcat": "^4.1.0",
```
Comment (dessalines) [inline package.json:8]: This didn't run without classcat.

**Comment by 0x241F31** [comment]
I think it could be better to switch from fusebox to rollup/webpack

**Comment by dessalines** [comment]
I'll try to make a pass at converting this to webpack, but I'm really bad at build systems.

**Pull request closed without merging**